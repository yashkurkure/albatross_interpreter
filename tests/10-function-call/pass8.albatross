while (f1()) {
  if (f2()) {
    f3();
  }
  repeat(f4() + f5() + 10 + a) {
    f6();
  }
}
