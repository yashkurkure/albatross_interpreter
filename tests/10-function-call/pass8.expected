f1()
f2()
f3()(((f4()+f5())+(10))+(a))
f6()
